### ISSUE TITLE
Unable to cast Rich Text Content

### ISSUE THREAD

**Issue by vivekdhiman** [contentful/rich-text-renderer-java#12]
Hi,

I have basically 2 issues here. 

- As per the documentation
```
final HtmlProcessor processor = new HtmlProcessor();
final HtmlContext context = new HtmlContext();
final String html = processor.render(context, node);
```
 Here, **render** method is not able to resolve when I am using these APIs.
- Getting class cast exception when trying to use CDARichDocument. Below are some screenshots related with this.
<img width="472" alt="Screen Shot 2020-04-19 at 3 26 25 PM" src="https://user-images.githubusercontent.com/7066652/79703635-3117a980-826a-11ea-971c-b76dff16e871.png">
<img width="547" alt="Screen Shot 2020-04-19 at 6 15 55 PM" src="https://user-images.githubusercontent.com/7066652/79703653-4096f280-826a-11ea-951e-7cd8dbe5cca1.png">
<img width="574" alt="Screen Shot 2020-04-19 at 6 19 43 PM" src="https://user-images.githubusercontent.com/7066652/79703674-6cb27380-826a-11ea-9237-fc6a043011ed.png">

**Comment by rafalniski** [comment]
Hi @vivekdhiman 

Indeed, there was an error in the docs, please use `process` method instead of `render`.
As per second issue, I'm not able to reproduce it. Could you please attach your code snippet?

**Comment by JamesJ** [comment]
Getting the same error when trying to process a rich text field:
```java
    private String fetchHtml() {
        Logger.getLogger().info("html=" + entry.getField("html"));
        CDARichDocument document = entry.getField("html");
        HtmlProcessor processor = new HtmlProcessor();
        HtmlContext context = new HtmlContext();

        return processor.process(context, document);
    }
```

Outputs a ClassCastException that `java.lang.ClassCastException: com.google.gson.internal.LinkedTreeMap cannot be cast to com.contentful.java.cda.rich.CDARichDocument`.
The logging line shows this:
`[2020-08-12 23:16:03] [INFO] html={data={}, content=[{data={}, content=[{data={}, marks=[], value=This is some text, nodeType=text}], nodeType=paragraph}, {data={}, content=[{data={}, marks=[], value=Hi! , nodeType=text}], nodeType=paragraph}], nodeType=document}
`

**Comment by rafalniski** [comment]
This should be fixed now in version `1.5.0`
